ocfweb/stats/printing.py

```diff
@@ -16,6 +16,7 @@
 
 
 ALL_PRINTERS = ('papercut', 'pagefault', 'logjam', 'deforestation')
+ACTIVE_PRINTERS = ('papercut', 'pagefault')
```

Comment:
what about reading these from ldap? e.g. like this:

```python
>>> import ocflib.infra.hosts
>>> {host['cn'][0] for host in ocflib.infra.hosts.hosts_by_filter('(type=printer)')}
{'papercut', 'pagefault'}
```

unfortunately we can't do this for old printers (they're not in ldap anymore), but it could work for current printers

(low priority since we'd still need to update ALL_PRINTERS anyway when we get a new printer)